store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+        event_data = create_cloudevent_json(project_id, database_id, collection_path, document_id, document_fields)
+        json_data = json.dumps(event_data, separators=(',', ':'))
+        
+        headers_str = ' \\\n  '.join(headers)
+        curl_cmd = f"""curl -X POST '{target_url}' \\
+  {headers_str} \\
+  -d '{json_data}'"""
+    
+    return curl_cmd
+
+
+def parse_fields(fields_str: str) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)